test_aodndata/aatams/test_aatams_sattag_qc_dm.py

```diff
@@ -0,0 +1,34 @@
+import os
+import unittest
+
+from aodncore.pipeline import PipelineFilePublishType, FileType
+from aodncore.testlib import HandlerTestCase
+from aodndata.aatams.aatams_sattag_qc_dm import AatamsSattagQcDmHandler
+
+TEST_ROOT = os.path.join(os.path.dirname(__file__))
+GOOD_ZIP = os.path.join(TEST_ROOT, 'simple_zip.zip')
+AATAMS_SATTAG_DM_BASE = "IMOS/AATAMS/AATAMS_SATTAG_QC_DM"
+
+
+class TestAatamsDmHandler(HandlerTestCase):
+    def setUp(self):
+        self.handler_class = AatamsSattagQcDmHandler
+        super(TestAatamsDmHandler, self).setUp()
+
+    def test_harvest_csvs_in_zip(self):
+        handler = self.run_handler(GOOD_ZIP)
+        for file in handler.file_collection:
+            if file.file_type is FileType.CSV:
+                self.assertEqual(file.publish_type,
+                                 PipelineFilePublishType.HARVEST_UPLOAD)
```

Comment:
Personally I would test the final outcome, rather than the intention, i.e.
```python
    self.assertTrue(file.is_harvested and file.is_stored)
```
(or `file.is_archived` below) See `PipelineFile` properties [here](https://github.com/aodn/python-aodncore/blob/841b1adf00f79f1378d99a9134413e4cce8641d8/aodncore/pipeline/files.py#L343).